docs/quickstart-powervm.md

```diff
@@ -0,0 +1,449 @@
+# Helper Node Quickstart Install
+
+This quickstart will get you up and running for HMC managed PowerVM.  You will need knowledge of the HMC to create the LPARs.
```

Comment:
@cs-zhang can you add a link to HMC and PowerVM for anyone wanting to learn more about it..

Reply:
will add link here to HMC docs